docs/Ladepunkte.md

```diff
@@ -1,7 +1,9 @@
-_Einstellungen -> Konfiguration -> Ladepunkte_
+# Einstellungen -> Konfiguration -> Ladepunkte
```

Comment:
Das Github-Wiki verwendet den Datei-Namen als Überschrift. Wenn wir in der Datei auch noch Überschriften haben, sind diese doppelt. Das kursive ist der Pfad zu den Einstellungen, die beschrieben werden, und nicht die Überschrift.